### ISSUE TITLE
Have you tried to train from scratch?

### ISSUE THREAD

**Issue by vtddggg** [bethgelab/game-of-noise#3]
Great work!
I see you are not using `torch.nn.DataParallel` but only have single GPU to finetune on pretrained `torchvision` weights.
Have you tried to train from scratch or using multi-gpu for training?

Thanks

**Comment by EvgeniaAR** [comment]
Hi there. 1) We haven't tried using multi-gpu for training. 2) We tried training from scratch for Gaussian Noise Training, but we did not see any benefits compared to fine-tuning and fine-tuning is faster ;). We haven't tried training from scratch for ANT.

**Comment by vtddggg** [comment]
Got it. Thanks for your response.

**Comment by vtddggg** [comment]
Sorry, another question.
ImageNet-C has 19 corruptions, however, most papers (including yours) only test on 15 corruptions. Why not use all types of corruptions?

**Comment by EvgeniaAR** [comment]
The ImageNet-C paper proposes using the "main" 15 corruptions as a test set. Since in ML, one often wishes to have a validation set, one should then use the remaining 4 corruptions to tune hyper-parameters etc. It is arguable though whether the remaining 4 corruptions are "different enough" to not cause overfitting.

**Comment by vtddggg** [comment]
I see. Many thanks for your help.

**Comment by peace-andjoy** [comment]
> Hi there. 1) We haven't tried using multi-gpu for training. 2) We tried training from scratch for Gaussian Noise Training, but we did not see any benefits compared to fine-tuning and fine-tuning is faster ;). We haven't tried training from scratch for ANT.

hello! may i ask how long does it take to train it by single gpu?
